### PR TITLE
fix: cdn url

### PR DESCRIPTION
**Pull request by tgroutars**
Removes `/` character in url that shouldn't be there.

### CODE REVIEW

File: packages/graphql-playground-html/src/render-playground-page.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ const loading = getLoadingMarkup()
 const CONFIG_ID = 'playground-config';
 
 const getCdnMarkup = ({ version, cdnUrl = '//cdn.jsdelivr.net/npm', faviconUrl }) => {
-  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}/${version ? `@${version}/` : ''}${suffix}` || '')
+  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}${version ? `@${version}/` : ''}${suffix}` || '')
```
Comment (hemal7735) [inline packages/graphql-playground-html/src/render-playground-page.ts:92]: Hi @tgroutars ,
I was about to raise a PR and then found your PR, thanks for raising the PR.

I'd like to provide a typo fix here in the sense that when a `version` is not there, we still need `/`.

```suggestion
  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}${version ? `@${version}/` : '/'}${suffix}` || '')
```

**Comment by hemal7735** [comment]
Hello @acao, could you help to review this PR?

File: packages/graphql-playground-html/src/render-playground-page.ts [hunk 2]
```diff
@@ -89,7 +89,7 @@ const loading = getLoadingMarkup()
 const CONFIG_ID = 'playground-config';
 
 const getCdnMarkup = ({ version, cdnUrl = '//cdn.jsdelivr.net/npm', faviconUrl }) => {
-  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}/${version ? `@${version}/` : ''}${suffix}` || '')
+  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}${version ? `@${version}/` : '/'}${suffix}` || '')
```
Comment (osher-sade) [inline packages/graphql-playground-html/src/render-playground-page.ts:92]: Hey, I know it doesn't really change anything, but I would suggest formatting the URL like so:

```suggestion
  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}${version ? `@${version}` : ''}/${suffix}` || '')
```

Comment (tgroutars) [reply packages/graphql-playground-html/src/render-playground-page.ts]: Yep that makes sense

**Comment by wyattjoh** [comment]
☝️ Umm, looks like this shouldn't have happened....

**Comment by jedwards1211** [comment]
This needs to be released

**Comment by jedwards1211** [comment]
I'm very confused, how is GraphQL Playground not broken for 100% of users right now because of this issue?

**Comment by acao** [comment]
hey folks! I'm not sure how this PR got closed either. looks like you can automatically close issues from other repos which doesnt seem like something that should happen

I've been maintaining this as a hobby effort, since i'm mostly focused on `graphiql` and `vscode-graphql` for work, and i just got back from vaction. We were supposed to freeze this repo a while ago, if you didn't see the article, but there are just never ending bugs. I'm gonna spend some time on it tonight and see if we can't get a few more fixes in.

**Comment by acao** [comment]
@jedwards1211 it's only broken if they specify version

**Comment by jedwards1211** [comment]
Yeah, I figured that out finally and opened an issue about it, and closed it once this got merged

**Pull request merged**